data/src/test/assets/api_mocks/feedback_reporting.json

```diff
@@ -2,13 +2,25 @@
   "api_key": "",
```

Comment:
This can be removed

Reply:
Done.

Reply:
Resolving on behalf of @jcqli  as the change has been approved.